src/pudl/output/ferc1.py

```diff
@@ -228,6 +228,55 @@ def _out_ferc1__yearly_plants_utilities(
     )
 
 
+## example/draft factory pattern
+specs = [
+    {
+        "name": "out_ferc1__yearly_purchased_power_and_exchanges_sched326",
+        "df1": "core_ferc1__yearly_purchased_power_and_exchanges_sched326",
+        "df2": "core_pudl__assn_ferc1_pudl_utilities",
```

Comment:
Note that in all cases, `df2` is the same. So it doesn't need to be part of these inputs. It can be hard-coded as one of the input assets that are used by the asset definition (I think it's used in all of the FERC 1 output assets)